lib/cassandra/protocol/coder.rb

```diff
@@ -254,7 +255,12 @@ def read_value_v4(buffer, type, custom_type_handlers)
         when :smallint         then read_smallint(buffer)
         when :time             then read_time(buffer)
         when :date             then read_date(buffer)
-        when :custom           then read_custom(buffer, type, custom_type_handlers)
+        when :custom
+          if type.name == "com.backupify.db.DatumType"
```

Comment:
You could make this a constant since it's referenced a few times. Doesn't really matter to me though.